lib/kaffe/group_member/manager/group_member_supervisor.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Kaffe.GroupMemberSupervisor do
+  @moduledoc """
+  The top-level supervisor for group members and subscribers.
+  """
+
+  use Supervisor
+
+  require Logger
+
+  def start_link do
+    Supervisor.start_link(__MODULE__, :ok)
+  end
+
+  def start_worker_supervisor(supervisor_pid) do
+    Supervisor.start_child(supervisor_pid, supervisor(Kaffe.WorkerSupervisor, []))
```

Comment:
This kind of passing along to another supervisor seems unusual for OTP. Why does this supervisor need to be the one that tells another supervisor to start a child? Should that knowledge be at a different level of abstraction?